packages/front/.gitignore

```diff
@@ -17,6 +17,11 @@
 .env.test.local
 .env.production.local
 
+.cache
+dist
+
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+.eslintcache
```

Comment:
@thesocialdev maybe add a newline here? :joy:

Reply:
I agree with Wellindinho, you can use [EditorConfig](https://editorconfig.org) to standardize this type of configuration (end of line, indentation).

And you are ignoring `.eslintcache` but it is already versioned. After this PR is accepted, it will be necessary to delete the file and version the deletion to .gitignore works.

Reply:
EditorConfig + eslint + prettier will come in another patch. Will just put the new line manually this time.